errbot/backends/slack.py

```diff
@@ -147,11 +147,7 @@ def aclattr(self):
     @property
     def fullname(self):
         """Convert a Slack user ID to their user name"""
-        user = self._sc.server.users.find(self._userid)
-        if user is None:
-            log.error("Cannot find user with ID %s" % self._userid)
-            return "<%s>" % self._userid
-        return user.real_name
+        return "@%s" % self.username
```

Comment:
This is incorrect, `fullname` should return the user's full name/real name, not their username (`.person` does that).